geps/gep-713/index.md

```diff
@@ -109,395 +114,67 @@ additional fields that are _not_ in the `spec` of the object.
 
 ### Direct Policy Attachment
 
-A Direct Policy Attachment is tightly bound to one instance of a particular
-Kind within a single namespace (or to an instance of a single Kind at cluster scope),
-and only modifies the behavior of the object that matches its binding.
+For more description of the details of Direct Policy Attachment,
+see [GEP-2648](https://gateway-api.sigs.k8s.io/geps/gep-2648/).
 
-As an example, one use case that Gateway API currently does not support is how
-to configure details of the TLS required to connect to a backend (in other words,
-if the process running inside the backend workload expects TLS, not that some
-automated infrastructure layer is provisioning TLS as in the Mesh case).
+### Inherited Policy Attachment
 
-A hypothetical TLSConnectionPolicy that targets a Service could be used for this,
-using the functionality of the Service as describing a set of endpoints. (It
-should also be noted this is not the only way to solve this problem, just an
-example to illustrate Direct Policy Attachment.)
+For more description of the details of Inherited Policy Attachment,
+see [GEP-2649](https://gateway-api.sigs.k8s.io/geps/gep-2649/).
 
-The TLSConnectionPolicy would look something like this:
+### How to determine if a Policy is a Direct or Inherited one
 
-```yaml
-apiVersion: gateway.networking.k8s.io/v1alpha2
-kind: TLSConnectionPolicy
-metadata:
-  name: tlsport8443
-  namespace: foo
-spec:
-  targetRef: # This struct is defined as part of Gateway API
-    group: "" # Empty string means core - this is a standard convention
-    kind: Service
-    name: fooService
-  tls:
-    certificateAuthorityRefs:
-      - name: CAcert
-    port: 8443
+The basic rule here is "Does the Policy affect _any_ other object aside from
+the one it targets?" If not, it's Direct. If so, it's inherited.
 
-```
+The reason for this is that Direct Attached Policies make it _much_ easier to
+understand the state of the system, and so can use a more simple `status` design.
+However, Inherited Policies require knowledge of more resources, and consequently
+a more complex status design.
```

Comment:
I think this belongs in the TLDR when explaining why there needs to be two different kind of policies.

Reply:
Great point, thanks. I added a paraphrase of this paragraph to the beginning of the document.